.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub.
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ["release"]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-rest-api-flint-example:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
Suggest change to `${{ github.repository_owner }}`. Refer docs here: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context